fenix/app/src/test/java/org/mozilla/fenix/home/HomeFragmentTest.kt

```diff
@@ -72,15 +73,18 @@ class HomeFragmentTest {
 
     @Test
     fun `GIVEN the selected search engine is set to eBay WHEN getTopSitesConfig is called THEN providerFilter filters the eBay provided top sites`() {
-        mockkStatic("mozilla.components.browser.state.state.SearchStateKt")
```

Comment:
this was actually failing some tests in other classes because it was not unmocked.